javascript_nodejs/14.nlp-with-dispatch/index.js

```diff
@@ -53,26 +53,25 @@ const adapter = new BotFrameworkAdapter({
     appPassword: endpointConfig.appPassword || process.env.microsoftAppPassword
 });
 
-// Define state store for your bot. See https://aka.ms/about-bot-state to learn more about bots memory service
-// A bot requires a sate store to priciest it dialog and user state between messages
+// Define state store for your bot. See https://aka.ms/about-bot-state to learn more about bot state.
 const memoryStorage = new MemoryStorage();
 // CAUTION: The Memory Storage used here is for local bot debugging only. When the bot
 // is restarted, anything stored in memory will be gone. 
-// For production bots use the Azure CosmosDB storage, Azure Blob, or Azure Table storage provides. 
+// For production bots use the Azure CosmosDB storage, Azure Blob storage provides. 
```

Comment:
grammer. [...] " Azure CosmosDB storage or Azure Blob storage providers". 
(add the or, and make provides = providers

Reply:
Fixed.